spec/omniauth/strategies/saml_spec.rb

```diff
@@ -222,4 +222,13 @@ def post_xml(xml=:example_response)
   it 'implements #on_metadata_path?' do
     expect(described_class.new(nil)).to respond_to(:on_metadata_path?)
   end
+
+  describe 'subclasses' do
+    let(:subclass) { Class.new(described_class) }
```

Comment:
Updated and amended. I also threw `described_class` into the `include` check since `omniauth-oauth2` is doing that check and it seems reasonable.